### DISCUSSION TITLE
Ability to have icons for column headers title

### DISCUSSION THREAD

**Discussion by akulmehta** [Power-Components/livewire-powergrid#516] [Ideas]
### Summary

Having icons for column headers can be useful to reduce the column width.

However, currently this is not possible as the string is escaped in output in `cols.blade.php`  with 

```blade
<span>{{ $column->title }}</span>
```

### Why is this needed?

Example, imagine if we have a column showing visibility as either private or public. Rather than showing the words "Private", "Public" and Visibility, we could have a more thinner column as seen in the screenshot below:

![image](https://user-images.githubusercontent.com/42812604/179401509-39035792-7e40-413c-8ab0-da5cb2073211.png)

### Suggested Solution(s)

There are two potential solutions for this.

1. Currently the column title is escaped in `cols.blade.php` using blade `{{ }}` double curly syntax. Since the column title is provided by the developer using either the `Column::make()` or `Column::title()` command, I am not sure if character escaping is required. 

`cols.blade.php:line33` change to:
```blade
<span>{!! $column->title !!}</span>
```

2. Expand the Column class and make methods to accept an optional $icon attribute which is rendered in the cols.blade.php `{!! !!}`.

### Related issue(s)/PR(s)

Happy to make a PR depending on which solution is better above.

**Comment by luanfreitasdev** [comment]
Thanks for your suggestion and I believe it to be a simple and effective solution. 

Can you send the PR ?

**Reply by akulmehta** [reply]
I will generate a PR. Thanks. Another positive side effect of option 1 is you can directly enter html as well. So for larger column titles one could split the title in two lines.
